src/types/config.ts

```diff
@@ -113,7 +137,83 @@ export const BotConfigSchema = Type.Object({
   comments: CommentsSchema,
   command: CommandConfigSchema,
   wallet: WalletSchema,
+  ask: AskSchema,
   accessControl: AccessControlSchema,
 });
 
 export type BotConfig = Static<typeof BotConfigSchema>;
+
+export const StreamlinedCommentSchema = Type.Object({
+  login: Type.Optional(Type.String()),
+  body: Type.Optional(Type.String()),
+});
+
+export type StreamlinedComment = Static<typeof StreamlinedCommentSchema>;
+
+export const GPTResponseSchema = Type.Object({
+  answer: Type.Optional(Type.String()),
+  tokenUsage: Type.Object({
+    output: Type.Optional(Type.Number()),
+    input: Type.Optional(Type.Number()),
+    total: Type.Optional(Type.Number()),
+  }),
+});
+
+export type GPTResponse = Static<typeof GPTResponseSchema>;
+
+export const WideConfigSchema = Type.Object(
```

Comment:
I never understood this "wide" prefix. I even tried removing it in my unmerged pull request from last month.

Reply:
Yeah me neither

Reply:
This should be refactored and more expressive. Refer to my open pull request.